radio/src/opentx.cpp

```diff
@@ -568,6 +568,11 @@ void checkBacklight()
       }
       if (backlightOn) {
         currentBacklightBright = requiredBacklightBright;
+#if defined(COLOR_LCD)
+        // force backlight on for color lcd radios
+        if(currentBacklightBright > BACLIGHT_LEVEL_MAX - BACKLIGHT_LEVEL_MIN)
+          currentBacklightBricht = BACKLIGHT_LEVEL_MAX - BACKLIGHT_LEVEL_MIN;
```

Comment:
`currentBacklightBricht`? Typo?

Reply:
Yes and los COLOR_LCD instead of COLORLCD, so the compiler did not find it.